cs50/2018/fall/challenges/site/check50/__init__.py

```diff
@@ -0,0 +1,8 @@
+from check50 import *
+
+
+class Site(Checks):
+
+    @check()
+    def submitted(self):
+        """site deployed to https://USERNAME.cs50.site"""
```

Comment:
"Site deployed to https://USERNAME.cs50.site/."

And be sure to force username to lowercase?